Airplane-tickets-UA.Data/Airport.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace Airplane_tickets_UA.Data
+{
+    public class Airport
+    {
+        public int Id { get; set; }
+        public ICAO ICAO { get; set; }
+        public string Name { get; set; }
+
+        public int CityId { get; set; }
```

Comment:
i guess there is no need to store both city Id and City itself, choose one from them

Reply:
I need both properties for further database samples. "CityId" is convenient for search queries, and "City" is for displaying data.

Reply:
but you could easily get one having another or at least have one of them private